testkit/src/main/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -1387,30 +1387,30 @@ protected static SortedSet<String> keywords(@Nullable String dialect) {
 
   @Test void testRowValueExpression() {
     final String expected0 = "INSERT INTO \"EMPS\"\n"
-            + "VALUES (ROW(1, 'Fred')),\n"
```

Comment:
why change these lines? This is not part of your PR. If the previous format is not right and you want to improve it, it may be better to submit a new PR.

Reply:
This seems to be a github display issue, I did not fix this part. I have also tried to modify it but it still shows, I will try again later

Reply:
I don't think so. you can rebase the latest main and have a try.